third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -692,7 +692,8 @@ impl Interpreter {
             err = new_err.finish(err.location().clone())
         }
         if err.status_type() == StatusType::InvariantViolation
-            && err.major_status() != StatusCode::STORAGE_ERROR
+            && err.major_status() != StatusCode::STORAGE_ERROR // should we core dump on storage error, now that aggregators have their own?
```

Comment:
Let's start logging storage errors properly (without speculative logging) to make sure we fully eliminate them first?